crates/bevy_ui/src/lib.rs

```diff
@@ -89,17 +89,17 @@ impl Plugin for UiPlugin {
                 CoreStage::PostUpdate,
                 flex_node_system
                     .label(UiSystem::Flex)
-                    .before(TransformSystem::TransformPropagate),
+                    .before(HierarchySystem::TransformPropagate),
```

Comment:
Mirroring my coments from another file. This seems backwards. TransformPropagate should remain a TransformSystem, and HierarchySystems for maintaining the hierarchy should be separate. Hopefully in the near future, we won't need any HierarchySystems.